posthog/api/hog_function.py

```diff
@@ -181,6 +182,8 @@ def to_internal_value(self, data):
         self.context["encrypted_inputs"] = instance.encrypted_inputs if instance else {}
 
         template = HogFunctionTemplates.template(data["template_id"]) if data["template_id"] else None
+        if not template:
+            capture_exception(Exception(f"No template found for id '{data['template_id']}'"))
```

Comment:
Can we add more tags here (teamId, functionId etc.) - otherwise debugging this will be impossible